deployments/production/spark-master-deployment.json

```diff
@@ -3,32 +3,22 @@
     "kind": "Deployment",
     "metadata": {
         "labels": {
-            "component": "spark-cluster-master",
```

Comment:
will these changes (and the corresponding changes in the spark 2-1-0 deployment) change the name of the dns resolution `spark-cluster-master` -> `spark-master`? 

I believe these are ultimately positive changes, but we will need to update job files as well

Reply:
This change specifically will not.  The labels only matter to the services selector (which I've also updated).  Changes to a service name _would_ impact the jobs, but those have not changed.  (I had thought about it, but reverted the service name changes due to the impact on job files, might revisit in the future though.)